ext/opentelemetry-ext-grpc/examples/helloworld_pb2_grpc.py

```diff
@@ -0,0 +1,46 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+import grpc
+
+import helloworld_pb2 as helloworld__pb2
+
+
+class GreeterStub(object):
```

Comment:
It is not necessary to inherit from `object` in Python 3.

Reply:
Sorry @ocelotl, these are generated files.